### DISCUSSION TITLE
Pesquisa sobre cursos online (focado em CSS)

### DISCUSSION THREAD

**Discussion by afonsopacifer** [frontendbr/forum#309] [Discussão]
Dando sequência ao meu plano maligno de evangelizar o CSS, 
preciso da ajuda de vocês para idealizar um novo projeto para a comunidade.

Quem tiver 3min livres :)

https://goo.gl/forms/RStdbItW6MctWKL12

![giphy](https://cloud.githubusercontent.com/assets/6748866/18848695/ec28fb14-8405-11e6-882f-5243c9f62fbb.gif)

**Comment by brunooomelo** [comment]
Minha resposta foi registrada. <3

**Comment by suissa** [comment]
Estamos esperando o de Animações de CSS FREEEE
